Makefile

```diff
@@ -39,13 +43,16 @@ BUNDLE_GEN_FLAGS ?= -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 # To enable set flag to true
 USE_IMAGE_DIGESTS ?= false
 ifeq ($(USE_IMAGE_DIGESTS), true)
-    BUNDLE_GEN_FLAGS += --use-image-digests
+	BUNDLE_GEN_FLAGS += --use-image-digests
 endif
 
+# Set the Operator SDK version to use. By default, what is installed on the system is used.
+# This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
+OPERATOR_SDK_VERSION ?= v1.38.0
```

Comment:
Could this be moved to the `## Tool Versions` section below, to keep all tools in one place?

Reply:
This is how operator-sdk scaffolds the Makefile. I see your point, but I'd prefer aligning to the standard scaffolding structure. In addition, your request could include also, perhaps, the `ENVTEST_K8S_VERSION` var. Moving those parts around could be confusing. Opinions?

Reply:
Ok, let's keep the operator-sdk scafffolding